sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1710,7 +1722,8 @@ class Analyzer(override val catalogManager: CatalogManager) extends RuleExecutor
         val resolvedNoOuter = partitionExprs.map(resolveExpressionByPlanChildren(_, r))
         val (newPartitionExprs, newChild) = resolveExprsAndAddMissingAttrs(resolvedNoOuter, child)
         // Outer reference has lower priority than this. See the doc of `ResolveReferences`.
-        val finalPartitionExprs = newPartitionExprs.map(resolveOuterRef)
+        val resolvedWithOuterRefs = newPartitionExprs.map(resolveOuterRef)
```

Comment:
```suggestion
        val resolvedWithOuter = newPartitionExprs.map(resolveOuterRef)
```

to match the existing naming style in this rule.